### PR TITLE
Fix/elements style polish

### PR DESCRIPTION
**Pull request by pkhlopiachyi**

### CODE REVIEW

**Comment by pkucherenk0** [comment]
QA Approved

File: src/containers/Confirm/Documents/index.tsx [hunk 1]
```diff
@@ -241,8 +241,16 @@ class DocumentsComponent extends React.Component<Props, DocumentsState> {
     };
 
     private handleFieldFocus = (field: string) => {
+
+        window.console.log(this.state.documentsFocused);
```
Comment (oyershov) [inline src/containers/Confirm/Documents/index.tsx]: please, remove this

File: src/containers/Confirm/Documents/index.tsx [hunk 2]
```diff
@@ -241,8 +241,16 @@ class DocumentsComponent extends React.Component<Props, DocumentsState> {
...
         return () => {
             switch (field) {
+                case 'document':
+                    window.console.log('kek');
```
Comment (oyershov) [inline src/containers/Confirm/Documents/index.tsx]: and this line too

File: src/screens/ProfileScreen/ProfileScreen.pcss [hunk 3]
```diff
@@ -557,6 +561,10 @@
             }
           }
         }
+
+        &:focus-within {
+
+        }
```
Comment (oyershov) [inline src/screens/ProfileScreen/ProfileScreen.pcss]: this is unused, remove

File: src/screens/ProfileScreen/ProfileScreen.pcss [hunk 4]
```diff
@@ -532,6 +532,10 @@
             legend {
               color: var(--accent) !important;
             }
+
+            label {
+              color: var(--accent) !important;
+            }
```
Comment (oyershov) [inline src/screens/ProfileScreen/ProfileScreen.pcss]: can be just:
```
legend, label {
    color: var(--accent) !important;
}
```

**Pull request merged**